steps/download-release-artifact.ps1

```diff
@@ -15,16 +15,15 @@ $RepoContext = "$OrgName/$RepoName"
 Write-Output  "Downloading the Artifact"
 gh release download --repo $RepoContext "$Version" --pattern "*.zip"
 
-# Unzip Artifact
-Write-Output  "Unzipping the Artifact"
-$ZipFiles = Get-ChildItem -Filter *.zip
-foreach ($file in $ZipFiles) {
-    Expand-Archive -Path $file.FullName -DestinationPath "./extracted-artifact"
-}
+# Write out the path to the artifact
+$artifactPath = Get-ChildItem -Path "./configurator-$Version" --pattern "*.zip"
```

Comment:
`--pattern` doesn't seem to be an option of `Get-ChildItem`, see: https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.management/get-childitem?view=powershell-7.4#syntax

Reply:
Yeah I copied the gh cli command by mistake. I have changed this to use -Filter now.